src/blog/2023/11/dashboard-2.0-user-tracking.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Tracking Who Has Opened a Dashboard"
+subtitle: "Using FlowFuse Authentication Audit Dashboard v2 Access"
+description: "Using FlowFuse Authentication Audit Dashboard v2 Access"
+date: 2023-11-09
+authors: ["ben-hardill"]
+image: "/blog/2023/11/images/dashboard-user-flow.png"
```

Comment:
While this works, it creates a fuzzy image for me

Reply:
I think you're correct, in DEV_MODE locally it yields different results compared to the Netlify deploy preview. Sorry about this, the image is fine indeed.